Stanford/Utility/Install/CreateMenuLinks.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file
+ * This function creates a new Menu Position rule.
+ */
+
+namespace Stanford\Utility\Install;
+
+/**
+ * Create new menu links.
+ * @param array $mp_rules A multidimensional array with the following keys:
```

Comment:
Move this doc block to the execute method.
Also, please update the variable names to match.